src/CreateX.sol

```diff
@@ -33,8 +33,8 @@ contract CreateX {
      * @dev Struct for the `payable` amounts in a deploy-and-initialise call.
      */
     struct Values {
-        uint256 constructorAmount;
-        uint256 initCallAmount;
+        uint128 constructorAmount;
+        uint128 initCallAmount;
```

Comment:
These are never written to storage so making them uint128 makes it more expensive to use, since ABI encoded calldata expands these to 32 bytes each anyway, so solidity will clean upper bits

Reply:
Right, change is reverted via https://github.com/pcaversaccio/createx/pull/22/commits/7d37a0369139d74df0472b42a7506b276412f279.